library/std/src/net/ip.rs

```diff
@@ -1282,7 +1282,7 @@ impl Ipv6Addr {
     /// +----------+-------------------------+----------------------------+
     /// ```
     ///
-    /// As a result, this method consider addresses such as `fe80:0:0:1::` or `fe81::` to be
+    /// As a result, this method considers addresses such as `fe80:0:0:1::` or `fe81::` to be
```

Comment:
Shouldn't this be `consider` since the word behind it is `addresses`?

Reply:
Why would that change things?

Reply:
I don't quite understand. Isn't the previous one correct and this one incorrect? Maybe I am wrong here.

Reply:
It's a third person singular. So needs an `s`. It does not belong to the exceptions like `can`, `must` etc.

Reply:
Oh you are correct. I was confused for a bit such that I was looking at `addresses` to determine if it need `s` instead of `this method`.